src/csv.js

```diff
@@ -19,7 +18,7 @@ async function ensure() {
                     'success',
                 ],
             })
-        if (!data.length) throw "fix"
+        if (!data.length) throw 'fix'
```

Comment:
Can we add more details to this assertion?

Reply:
Done